src/plugins/profiling/server/routes/search_flamechart.ts

```diff
@@ -90,25 +94,31 @@ export function getSampledTraceEventsIndex(
     for (let i = initialExp + 1; i <= maxExp; i++) {
       sampleCountFromInitialExp /= samplingFactor;
       if (sampleCountFromInitialExp < samplingFactor * targetSampleSize) {
-        return { name: downsampledIndex + i, sampleRate: 1 / samplingFactor ** i };
+        return { name: downsampledIndex(i), sampleRate: 1 / samplingFactor ** i };
       }
     }
     // If we come here, it means that the most sparse index still holds too many items.
     // The only problem is the query time, the result set is good.
-    return { name: downsampledIndex + 11, sampleRate: 1 / samplingFactor ** maxExp };
+    return { name: downsampledIndex(11), sampleRate: 1 / samplingFactor ** maxExp };
   } else if (sampleCountFromInitialExp < targetSampleSize) {
     // Search in less down-sampled indexes.
     for (let i = initialExp - 1; i >= 1; i--) {
       sampleCountFromInitialExp *= samplingFactor;
       if (sampleCountFromInitialExp >= targetSampleSize) {
-        return { name: downsampledIndex + i, sampleRate: 1 / samplingFactor ** i };
+        return {
```

Comment:
Interesting, is this ESLint'ed formatting ? It seems to behave slightly differently on my system.

Reply:
yep this is ESLint... i guess it changed because we're now calling a function vs adding strings??
No idea TBH